test/e2e/integration/001-hello-gutenberg.js

```diff
@@ -8,4 +8,30 @@ describe( 'Hello Gutenberg', () => {
 		cy.url().should( 'include', 'post-new.php' );
 		cy.get( '[placeholder="Add title"]' ).should( 'exist' );
 	} );
+
+	it( 'Should have no history', () => {
+		cy.get( '.editor-history__undo:not( :disabled )' ).should( 'not.exist' );
+		cy.get( '.editor-history__redo:not( :disabled )' ).should( 'not.exist' );
+	} );
+
+	it( 'Should not prompt to confirm unsaved changes', ( done ) => {
```

Comment:
I was afraid this was a false positive because I thought Cypress would remove all popovers but confirmed it's breaking correctly when needed.